packages/@ourworldindata/grapher/src/stackedCharts/StackedUtils.test.ts

```diff
@@ -29,26 +33,43 @@ const seriesArr = [
     },
 ]
 
-it("can add fake points", () => {
-    expect(seriesArr[1].points[1]).toEqual(undefined)
-    const series = withMissingValuesAsZeroes(seriesArr)
-    expect(series[1].points[1].position).toEqual(2002)
+describe(withUniformSpacing, () => {
+    it("can add values to make an array evenly spaced", () => {
+        expect(withUniformSpacing([])).toEqual([])
+        expect(withUniformSpacing([5])).toEqual([5])
+        expect(withUniformSpacing([5, 10])).toEqual([5, 10])
```

Comment:
There's one more test case I would like. because that shows that we look at deltas and not at the actual numbers:
```ts
expect(withUniformSpacing([7, 12, 17])).toEqual([7, 12, 17])
```